docs/using.adoc

```diff
@@ -208,41 +209,24 @@ OpenShift.
 === Using `docker`
 
 ADB provides a full container environment and runs both `docker` and
-`kubernetes`. All standard commands work, for example:
-
-....
-ADB pull centos
- docker run -t -i centos /bin/bash
-....
+`kubernetes`, with most standard commands. 
 
 [[using-atomic-app-and-nulecule]]
 === Using Atomic App and Nulecule
 
-Details on these projects can be found at these URLs:
+You can use Atomic App and Nulecule to run ADB. Details on these projects can be
+found at:
 
 * Atomic App: https://github.com/projectatomic/atomicapp
 * Nulecule: https://github.com/projectatomic/nulecule
 
-The
-https://registry.hub.docker.com/u/projectatomic/helloapache/[helloapache]
-example can be used to test your installation.
-
 NOTE: Many Nulecule examples expect a working Kubernetes environment.
 Use the
 link:../components/centos/centos-k8s-singlenode-setup/Vagrantfile[Vagrantfile]
 and refer the corresponding
 link:../components/centos/centos-k8s-singlenode-setup/README.adoc[README]
 to set up a single node Kubernetes environment.
 
-You can verify your environment by executing `kubectl get nodes`. The
-expected output is:
-
-....
-$ kubectl get nodes
-NAME        LABELS                             STATUS
-127.0.0.1   kubernetes.io/hostname=127.0.0.1   Ready
-....
-
 [[vagrant-bi-directional-folder-sync]]
```

Comment:
@thatdocslady any specific reason we are removing this section. Is it just to make it even for other services.